app/src/main/java/com/sunasterisk/travelapp/ui/adapter/RestaurantTabAdapter.kt

```diff
@@ -0,0 +1,56 @@
+package com.sunasterisk.travelapp.ui.adapter
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.sunasterisk.travelapp.R
+import com.sunasterisk.travelapp.data.models.Location
+import com.sunasterisk.travelapp.ui.base.BaseRecyclerAdapter
+import com.sunasterisk.travelapp.ui.base.BaseViewHolder
+import com.sunasterisk.travelapp.ui.adapter.RestaurantTabAdapter.LocationViewHolder
+import com.sunasterisk.travelapp.utils.setImageUrl
+import kotlinx.android.synthetic.main.item_destination.view.*
+
+class RestaurantTabAdapter(private var onItemClick: (Location) -> Unit) :
+    BaseRecyclerAdapter<Location, LocationViewHolder>() {
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): LocationViewHolder {
+        val itemView =
+            LayoutInflater.from(parent.context).inflate(R.layout.item_destination, parent, false)
```

Comment:
```suggestion
        val itemView = LayoutInflater.from(parent.context)
            .inflate(R.layout.item_destination, parent, false)
```
Có thể sửa hoặc k nhé